nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -101,6 +101,8 @@ ParameterHandler::ParameterHandler(
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".use_collision_detection",
     rclcpp::ParameterValue(true));
+  declare_parameter_if_not_declared(
+      node, plugin_name_ + ".stateful", rclcpp::ParameterValue(false));
```

Comment:
Make sure to update docs.nav2.org's configuration guide for RPP for this new parameter.

Also, should we potentially make this default true? Seems reasonable.

Reply:
https://github.com/ros-navigation/docs.nav2.org/pull/690